druid/src/core.rs

```diff
@@ -389,34 +361,34 @@ impl<T, W: Widget<T>> WidgetPod<T, W> {
     ///
     /// The provided `child_state` should be merged up if this returns `true`.
     fn set_hot_state(
```

Comment:
This comment needs to be rewritten given that both `rect` and `child_state` are no longer parameters.